packages/workflow/src/features/registration/fhir/fhir-bundle-modifier.ts

```diff
@@ -205,166 +202,3 @@ export function updatePatientIdentifierWithRN(
     return patient
   })
 }
-
-interface Integration {
-  name: string
-  status: string
-  integratingSystemType: 'MOSIP' | 'OTHER'
-}
-
-const statuses = {
-  PENDING: 'pending',
-  ACTIVE: 'active',
-  DISABLED: 'disabled',
-  DEACTIVATED: 'deactivated'
-}
-
-export async function validateDeceasedDetails(
-  patient: Saved<Patient>,
-  authHeader: { Authorization: string }
-): Promise<Saved<Patient>> {
-  /*
-    In OCRVS-1637 https://github.com/opencrvs/opencrvs-core/pull/964 we attempted to create a longitudinal
-    record of life events by an attempt to use an existing person in gateway if an identifier is supplied that we already
-    have a record of in our system, rather than creating a new patient every time.
-
-    However this supplied identifier cannot be trusted. This could lead to links between persons being abused or the wrong indivdual
-    being marked as deceased.
-
-    Any external identifier must be justifiably verified as authentic by a National ID system such as MOSIP or equivalent
```

Comment:
This premise can be handled within the `confirmRegistration` / `rejectRegistration` API endpoints. The MOSIP-API can check if the the NID supplied for the death is valid via MOSIP, and then either confirm or reject.